app/renderer/components/navbar_menu.js

```diff
@@ -10,12 +10,12 @@ module.exports = {
     template: `
     <ul id='navbar-menu' class='dropdown-content'>
         <template v-if="!editMode">
-            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'run-suite')">Run Suite</a></li>
-            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'stop-suite')">Stop Suite</a></li>
+            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'run-suite')" v-bind:href="'#tab0'">Run Suite</a></li>
```

Comment:
Do we really need `v-bind:href="'#tab`? I think there are easier native ways to achieve tabbing

Reply:
A simple `href` might work.

Reply:
I'm not sure you even need href, I think there is a tabable syntax or something like this